qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -219,94 +219,16 @@ admin_daemon()
     ceph --admin-daemon ${asok_file} $@
 }
 
-status()
```

Comment:
Then also need to remove `status` command from the example section in the beginning of the script.